### PR TITLE
resend lab3

### PR DESCRIPTION
**Pull request by Sosnin-Artur**

### CODE REVIEW

File: chat.html [hunk 1]
```diff
@@ -14,10 +14,11 @@ <h1> Messenger</h1>
     <main class="chat-main">
       <div class="chat-sidebar">
         <h3>Room Name:</h3>
-        <h2 id="room-name">RoomName</h2>
+        <h3 id="room-name">RoomName</h3>
```
Comment (katetushkan) [inline chat.html:17]: u can't use `<h3>` without using `<h2>`

Comment (katetushkan) [reply chat.html:17]: I think also, that u forget to redeploy.

**Review by katetushkan — CHANGES_REQUESTED** [review]
Where is CSS ?
There is no mobile version
Screenshot 2022-04-18 at 23 42 07
(

**Comment by Sosnin-Artur** [comment]
deploy address: https://checking-lab.herokuapp.com/

File: css/style.css [hunk 2]
```diff
@@ -6,48 +6,53 @@
...
+* {	
 	box-sizing: border-box;
 	margin: 0;
 	padding: 0;
 }
 
 body {
 	font-family: 'Roboto', sans-serif;
-	font-size: 16px;
+	font-size: 3vw;
```
Comment (katetushkan) [inline css/style.css:17]: That's a bad practice. Font size can be the same. You need to adapt element and layout dimensions instead.

File: css/style.css [hunk 3]
```diff
@@ -6,48 +6,53 @@
...
 	background: var(--light-color);
-	margin: 20px;
+	margin: 4%;
 }
 
 .btn {
 	cursor: pointer;
-	padding: 5px 15px;
+	padding: 1% 3% ;
```
Comment (katetushkan) [inline css/style.css:24]: Percentage values for margins and paddings are also not a good idea.

File: css/style.css [hunk 4]
```diff
@@ -6,48 +6,53 @@
...
 /* Chat Page */
 
 .chat {
-	max-width: 1100px;
+	width: 100%;
+	height: 100%;
 	background: #fff;
-	margin: 30px auto;
-	overflow: hidden;
+	margin: center;
```
Comment (katetushkan) [inline css/style.css:37]: Hm?

**Review by katetushkan — CHANGES_REQUESTED** [review]
image

This is not a mobile version. You just scaled down the website. Now try to hit buttons or inputs with your finger. It will be pretty tough task for you I'm sure.

Mobile version assumes your UI is adapted for smaller screen sizes. Take a look at mobile apps and mobile version of websites (take Twitter as example).

Drop percentages, drop `vw` units.

**Review by katetushkan — APPROVED** [review]
<kbd>8</kbd>

**Review by katetushkan — APPROVED** [review]

**Pull request closed without merging**